packages/pds/tests/views/admin/get-moderation-reports.test.ts

```diff
@@ -241,6 +242,22 @@ describe('pds admin get moderation reports view', () => {
     expect(forSnapshot(reportsWithTakedown.data.reports)).toMatchSnapshot()
   })
 
+  it('gets all moderation reports actioned by a certain moderator.', async () => {
+    const [actionedByAdminOne, actionedByAdminTwo] = await Promise.all([
+      agent.api.com.atproto.admin.getModerationReports(
+        { actionedBy: 'did:example:admin0' },
+        { headers: { authorization: adminAuth() } },
+      ),
+      agent.api.com.atproto.admin.getModerationReports(
+        { actionedBy: 'did:example:admin2' },
+        { headers: { authorization: adminAuth() } },
+      ),
+    ])
+
+    expect(forSnapshot(actionedByAdminOne.data.reports)).toMatchSnapshot()
+    expect(forSnapshot(actionedByAdminTwo.data.reports)).toMatchSnapshot()
```

Comment:
Since `forSnapshot` normalizes the did values away, it might just be worth an additional check that each item in `actionedByAdminOne.data.reports` has the expected did in `createdBy`.

Reply:
we don't include `createdBy` for actions in `getModerationReports` so I had to call `getModerationReport` with individual report id to check this. not sure if that breaks some kind of convention we follow for these tests @devinivy